lib/controller/controller.go

```diff
@@ -473,8 +511,8 @@ func NewProvisionController(
 
 	volumeHandler := cache.ResourceEventHandlerFuncs{
 		AddFunc:    nil,
```

Comment:
Why omitting `AddFunc` here? If omitted, volumes will not be queued until first resync happens (15 minutes later).

Reply:
we only care about volume Updates and not Adds because we are checking if a volume has entered 'Released' state and needs to be deleted

Reply:
but volumes entered 'Released' state during provisioner is not running (e.g. restarting or migrating to another node) will be delayed 15 minutes to delete.  I'm not sure if this is acceptable. Old controller implementation omitted `Add` handler also, but its resync period is only 15 seconds.

`shouldDelete` checks volume itself, queuing all volumes should not affect performance much IMO.

Reply:
Nice, you're correct, I've added AddFunc.